frappe/www/website_script.js

```diff
@@ -1,6 +1,41 @@
 // website_script.js
 {% if javascript -%}{{ javascript }}{%- endif %}
 
+{% if show_cookie_consent -%}
+frappe.require([
+	'/assets/frappe/node_modules/cookieconsent/build/cookieconsent.min.js',
```

Comment:
Do we really need a 20kb JS lib for showing a cookie consent popup?

Reply:
It does lot's of things: theming, different locations for the overlay, different types of consent (opt-in, opt-out, info, categories), etc. I wouldn't like to write that from scratch.